### ISSUE TITLE
Error in Swift 4.2

### ISSUE THREAD

**Issue by MrMatten** [ridvank/RKPieChart#6]
Any chance to get it updated to the latest version of swift?
It is a really nice package

**Comment by Yonodactyl** [comment]
Can you specify what your error is? This issue cant be resolved without direction.

**Comment by MrMatten** [comment]
Yes of course, sorry seemed like a small mistake that would come automatically, but I can't of course be sure of that
![image](https://user-images.githubusercontent.com/15010757/46196865-6cf5c180-c308-11e8-9fc3-7e435baa4450.png)

It happened straight after I updated. I believe it is Swift 4.2, but I am not sure

**Comment by davcpas1234** [comment]
to resolve this you need to change
```
shapeLayer.lineCap = lineCap
```
to:
```
shapeLayer.lineCap = CAShapeLayerLineCap(rawValue: lineCap)
```

**Comment by MrMatten** [comment]
I did locally, but prefer not to edit packages

**Comment by davcpas1234** [comment]
I know, I prefer not to either, but we're stuck unless the repo owner updates it

**Comment by vipinlelab** [comment]
Any update on this mentioned issue?
